### PR TITLE
Update new expression2 file comment

### PR DESCRIPTION
**Pull request by deltamolfar**
Makes it a little more modern.
Can be used to tell players that don't sit on Discord/don't follow GitHub updates what's new.
Informs regulars players about events/lambdas.

### CODE REVIEW

**Comment by wrefgtzweve** [comment]
I feel like updates shouldn't be in there as it'll likely rarely be updated.

**Comment by deltamolfar** [comment]
If so, it can be renamed to latest features, to inform about latest big stuff

**Comment by wrefgtzweve** [comment]
My point still stands then, a string in a file somewhere isn't a good way to display new features and will simply age poorly.

File: lua/wire/client/text_editor/wire_expression2_editor.lua [hunk 1]
```diff
@@ -959,16 +959,26 @@ function Editor:InitComponents()
...
+    - Compiler rewritten by Vurv
+
+    Documentation, instructions and examples are available at:
     https://github.com/wiremod/wire/wiki/Expression-2
-    ^ Read what @strict and other directives do here ^
+    ^ There you can read about: ^
+
+    - Why you should use @strict
+    - Other directives(like @model)
+    - What is events, and their benefits
```
Comment (Periapsises) [inline lua/wire/client/text_editor/wire_expression2_editor.lua:975]:
```suggestion
    - What are events and their benefits
```

File: lua/wire/client/text_editor/wire_expression2_editor.lua [hunk 2]
```diff
@@ -959,16 +959,26 @@ function Editor:InitComponents()
...
+    - _TAU bug fixed
+    - Compiler rewritten by Vurv
+
+    Documentation, instructions and examples are available at:
     https://github.com/wiremod/wire/wiki/Expression-2
-    ^ Read what @strict and other directives do here ^
+    ^ There you can read about: ^
+
+    - Why you should use @strict
+    - Other directives(like @model)
```
Comment (Periapsises) [inline lua/wire/client/text_editor/wire_expression2_editor.lua:974]:
```suggestion
    - Other directives (like @model)
```

File: lua/wire/client/text_editor/wire_expression2_editor.lua [hunk 3]
```diff
@@ -959,16 +959,26 @@ function Editor:InitComponents()
...
+    - What is events, and their benefits
+    - What is lambda functions, and their benefits
```
Comment (Periapsises) [inline lua/wire/client/text_editor/wire_expression2_editor.lua:976]:
```suggestion
    - What are lambda functions and their benefits
```

**Review by Periapsises — COMMENTED** [review]
Nice overall, but some grammar mistakes here and there.

**Comment by Periapsises** [comment]
> My point still stands then, a string in a file somewhere isn't a good way to display new features and will simply age poorly.

I don't think a lot of people will remember to include their changes in there.
If there was a way to automate it, that would be great.

**Comment by deltamolfar** [comment]
Thanks @Periapsises, I commited them

**Comment by Denneisk** [comment]
I think the changelog is going to be more trouble than it's worth. I'd like an in-editor changelog feature as well but I haven't thought on how to best approach such a thing.

The phrase "and their benefits" seems a little tacky in my opinion. :P

**Comment by deltamolfar** [comment]
Changelog - well tbh no idea, you lads can keep it or remove it, I don't really mind.

But "And their benefits" is for those that doesn't want to learn new things, and switch to better ways of doing things.

**Comment by deltamolfar** [comment]
Regarding news, I think separate file with new and old news can easily be added, which later on could be used to show news, with an inbox icon (similar to credits icon) with a read/not read support. Code wise it would just be a panel with static text from the said file

**Review by thegrb93 — CHANGES_REQUESTED** [review]
A changelog should be read from something else that regularly gets updated.
Too much info about the docs.

**Comment by deltamolfar** [comment]
Well, it can be done like that. To be honest I don't particulary care about the description format as much, as about it being informative about concepts that aren't easily discoverable by E2Helper (lambdas, events, and why @strict) mainly

**Review by Denneisk — APPROVED** [review]

**Review by thegrb93 — APPROVED** [review]

**Pull request merged**